src/components/reference-book/page.cjsx

```diff
@@ -13,7 +13,7 @@ GetPositionMixin = require '../get-position-mixin'
 {ReferenceBookStore} = require '../../flux/reference-book'
 {ReferenceBookExerciseStore} = require '../../flux/reference-book-exercise'
 
-EXERCISE_MATCHER = new RegExp('#ost\/api\/ex\/(.*)')
+EXERCISE_MATCHER = new RegExp('\/exercises\/(.*)')
```

Comment:
I just had to change this regexp in https://github.com/openstax/tutor-js/commit/880fe0ea6a032f45b1e5098704ebb4705a130df6 switching to use the CSS selector